Upstream: Unknown
Source: Gentoo
Reason: allow ipp to work over https

http://bugs.gentoo.org/show_bug.cgi?id=217293
original patch by Timo Gurr

diff -Naur cups-1.3.7/backend/Makefile cups-1.3.7.new/backend/Makefile
--- cups-1.3.7/backend/Makefile	2007-08-08 21:27:51.000000000 +0200
+++ cups-1.3.7.new/backend/Makefile	2008-04-14 16:42:18.106785330 +0200
@@ -62,6 +62,7 @@
 	done
 	$(RM) $(SERVERBIN)/backend/http
 	$(LN) ipp $(SERVERBIN)/backend/http
+	$(LN) ipp $(SERVERBIN)/backend/https
	if test "x$(DNSSD_BACKEND)" != x; then \
		$(RM) $(SERVERBIN)/backend/mdns; \
		$(LN) $(DNSSD_BACKEND) $(SERVERBIN)/backend/mdns; \
